plugins/scaffolder-backend/src/scaffolder/actions/builtin/fetch/cookiecutter.ts

```diff
@@ -86,12 +100,18 @@ export function createFetchCookiecutterAction(options: {
 
       const cookiecutter = templaters.get('cookiecutter');
 
+      const values = {
+        ...(ctx.input.values as TemplaterValues),
+        copyWithoutRender: ctx.input.copyWithoutRender,
```

Comment:
Should be `_copy_without_render` here right?

Reply:
Ah, good catch.

Reply:
Fixed in: https://github.com/backstage/backstage/pull/4956/commits/f4698d96543af572c51a32b32d94103681568bf3